MMOCoreORB/src/server/zone/objects/creature/commands/HealEnhanceCommand.h

```diff
@@ -438,8 +438,14 @@ class HealEnhanceCommand : public QueueCommand {
 		}
 
 		PlayerManager* playerManager = server->getZoneServer()->getPlayerManager();
+		int duration = enhancePack->getDuration();
```

Comment:
This [needs to be a float](https://github.com/swgintended/Core3/blob/31777f7f2df46ff47d418a541b4dac8a5b606a86/MMOCoreORB/src/server/zone/objects/tangible/pharmaceutical/EnhancePack.idl#L134) so the duration doesn't get truncated